core/storage/file/gae_models.py

```diff
@@ -83,27 +157,54 @@ class FileSnapshotMetadataModel(base_models.BaseSnapshotMetadataModel):
 
 
 class FileSnapshotContentModel(base_models.BaseSnapshotContentModel):
-    """Class for storing the content of the file snapshots."""
+    """Class for storing the content of the file snapshots.
+
+    Attributes:
+        content: BlobProperty. The contents of a file snapshot.
```

Comment:
Can drop this -- see above. (But I suggest augmenting the comment above the field.)